CMakeLists.txt

```diff
@@ -27,16 +28,83 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 project(chimera-sdk LANGUAGES C ASM)
 
+# WIESEP: It is important to set the ISA and ABI for the host and the cluster snitch
+set(ABI ilp32)
+set(ISA_CLUSTER_SNITCH rv32im)
+set(ISA_HOST rv32imc)
```

Comment:
I think we can tie this up one level lower - I'm thinking this might want to live in `targets`.

Reply:
Yes, I agree. I will change this.